translatorsinfo/translatorsinfo.cpp

```diff
@@ -353,11 +354,11 @@ TranslatorsInfo::TranslatorsInfo()
 {
     //fillLangguages(&mLanguagesList);
 
-    QSettings src(QStringLiteral(":/translatorsInfo"), QSettings::IniFormat);
-    src.setIniCodec("UTF-8");
+    QSettings src(QStringLiteral(":/translatorsInfo"), QSettings::Format::IniFormat);
+    //Format("UTF-8");not working yet
```

Comment:
Maybe not even needed?

Reply:
Please keep the first line exactly as it was (it doesn't need any change) :

```c++
    QSettings src(QStringLiteral(":/translatorsInfo"), QSettings::IniFormat);
```

And just remove `src.setIniCodec("UTF-8");` below it — in Qt6, everything is in UTF-8.